lib/engine/game/g_18_ms.rb

```diff
@@ -200,10 +200,31 @@ def or_set_finished
       end
 
       def or_description_short(turn, round)
+        G18MS.or_description_short(turn, round)
+      end
+
+      def self.or_description_short(turn, round)
         ((turn - 1) * 2 + round).to_s
       end
 
-      # Game will end directly after the end of OR 10
+      def self.game_card_round_info(gdata)
```

Comment:
game engines shouldn't take in gdata

Reply:
Ok. I will pass the values on line 211-213 as parameters instead.